File: 260120/살 수 있는 물건/things-able-to-buy.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        int book_price=3000;
        int mask_price=1000;

        Scanner sc=new Scanner(System.in);
        int wallet=sc.nextInt();

        if(wallet>=book_price){
            System.out.println("book");
        }
        else if(wallet>=mask_price){
            System.out.println("mask");
        }
        else{
            System.out.println("no");
        }
    }
}

File: 260223/살 수 있는 물건/things-able-to-buy.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int n;
    cin>>n;

    int book=3000;
    int mask=1000;

    if(n>=book){
        cout<<"book";
    }
    else if(n>=mask){
        cout<<"mask";
    }
    else{
        cout<<"no";
    }
    return 0;
}